### PR TITLE
add configurable implementation of shared header and footer

### PR DESCRIPTION
**Pull request by whitfield-mj**

### CODE REVIEW

File: server/views/partials/header.njk [hunk 1]
```diff
@@ -1,33 +1,45 @@
...
         </svg>
-        HMPPS
+        Digital Prison Services
```
Comment (andrewrlee) [inline server/views/partials/header.njk:18]: This might be a step too far

File: assets/scss/components/_footer.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.govuk-footer {
+  min-height: 160px;
+}
```
Comment (psoleckimoj) [inline assets/scss/components/_footer.scss:3]: Missing new line at end of the file.

File: assets/scss/components/_header-bar.scss [hunk 3]
```diff
@@ -114,28 +120,7 @@
...
-    @include govuk-link-style-default;
-    @include govuk-font($size: 19, $weight: 'normal');
-  }
-
-  @media print {
-    display: none;
-  }
-}
+  border: none;
+}
```
Comment (psoleckimoj) [inline assets/scss/components/_header-bar.scss:126]: Missing new line

File: server/views/partials/footer.njk [hunk 4]
```diff
@@ -0,0 +1 @@
+<footer class="govuk-footer govuk-!-display-none-print"></footer>
```
Comment (psoleckimoj) [inline server/views/partials/footer.njk:1]: Missing new line

File: assets/scss/components/_footer.scss (hunk 2 above)
Comment (williamfalconeruk) [inline assets/scss/components/_footer.scss:3]:
```suggestion
}

```

**Comment by petergphillips** [comment]
Closing due to comment by @whitfield-mj 
> Hi Jim. It was decided not to merge it in because the template is used by non-dps apps. It is on the roadmap to make implementation easier via an npm package to import, but unfortunately that hasn't been worked on yet

**Pull request closed without merging**